src/huggingface_hub/keras_mixin.py

```diff
@@ -152,22 +152,24 @@ def save_pretrained_keras(
     task_name: Optional[str] = None,
     **model_save_kwargs,
 ):
-    """Saves a Keras model to save_directory in SavedModel format. Use this if you're using the Functional or Sequential APIs.
-
-    model (`Keras.Model`):
-        The Keras model you'd like to save. The model must be compiled and built.
-    save_directory (`str`):
-        Specify directory in which you want to save the Keras model.
-    config (`dict`, *optional*):
-        Configuration object to be saved alongside the model weights.
-    include_optimizer(`bool`, *optional*, defaults to `False`):
-        Whether or not to include optimizer in serialization.
-    plot_model (`bool`, *optional*, defaults to `True`):
-        Setting this to `True` will plot the model and put it in the model card. Requires graphviz and pydot to be installed.
-    task_name (`str`, *optional*):
-        Name of the task the model was trained on. See the available tasks at https://github.com/huggingface/hub-docs/blob/main/js/src/lib/interfaces/Types.ts.
-    model_save_kwargs(`dict`, *optional*):
-        model_save_kwargs will be passed to tf.keras.models.save_model().
+    """
+    Saves a Keras model to save_directory in SavedModel format. Use this if you're using the Functional or Sequential APIs.
+
+    Args:
+        model (`Keras.Model`):
+            The Keras model you'd like to save. The model must be compiled and built.
+        save_directory (`str`):
+            Specify directory in which you want to save the Keras model.
+        config (`dict`, *optional*):
+            Configuration object to be saved alongside the model weights.
+        include_optimizer(`bool`, *optional*, defaults to `False`):
+            Whether or not to include optimizer in serialization.
+        plot_model (`bool`, *optional*, defaults to `True`):
+            Setting this to `True` will plot the model and put it in the model card. Requires graphviz and pydot to be installed.
+        task_name (`str`, *optional*):
+            Name of the task the model was trained on. See the available tasks at https://github.com/huggingface/hub-docs/blob/main/js/src/lib/interfaces/Types.ts.
```

Comment:
the link could be a `Reference`, but it should also be in the docs and not on github I think.